code/game/objects/items/tools/weldingtool.dm

```diff
@@ -255,7 +255,7 @@
 	. = ..()
 	if(. && user)
 		if (progress_flash_divisor == 0)
-			user.flash_act(min(light_range,1))
+			user.flash_act(min(light_range, 2))
```

Comment:
Why is this being clamped anyways?

Reply:
I have no clue

Reply:
It's likely to make sure it doesn't somehow end up being 0

Reply:
then it should be `max()`